legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -212,3 +213,171 @@ def search_businesses():
         current_app.logger.info(err)
         current_app.logger.error('Error searching over business information for: %s', identifiers)
         return {'error': 'Unable to retrieve businesses.'}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+# experimental debugging endpoints
```

Comment:
The following endpoints are for scenario testings. Singular form (e.g. `insert`) indicates that only one record is modified and plural form (e.g. `inserts`) indicate that multiple FK related records are updated.